core/arch/aarch64/codec.txt

```diff
@@ -257,7 +258,7 @@ x0110111xxxxxxxxxxxxxxxxxxxxxxxx  tbnz   tbz
 ## Unconditional branch (register)
 
 1101011000011111000000xxxxx00000  br     : x5
-1101011000111111000000xxxxx00000  blr    : x5
+1101011000111111000000xxxxx00000  blr    impx30: x5
```

Comment:
Unfortunately we can't do that without rewriting parts of the encoder/decoder generator. 

For `bl     b`,  a special `encode_opnds_b` function is used to encode all operands. This function takes the current PC as an argument, which `encode_opnds_b` uses to calculate the PC relative address used by `bl`. The same function is used to encode other instructions (without implicit operands) that use PC relative addressing as well.

I hope that makes sense.